tests/lib/prepare-project.sh

```diff
@@ -141,14 +175,23 @@ case "$SPREAD_SYSTEM" in
 esac
 
 # update vendoring
-if [ "$(which govendor)" = "" ]; then
-    rm -rf "$GOHOME/src/github.com/kardianos/govendor"
+if [ -z "$(which govendor)" ]; then
+    rm -rf $GOPATH/src/github.com/kardianos/govendor
     go get -u github.com/kardianos/govendor
 fi
 quiet govendor sync
 
 if [ -z "$SNAPD_PUBLISHED_VERSION" ]; then
-    build_deb
+    case "$SPREAD_SYSTEM" in
+      ubuntu-*|debian-*)
+         build_deb
+         ;;
+      fedora-*)
+         fedora_build_rpm
+         ;;
+      *)
```

Comment:
Again, we don't need that if it's a passthrough I think.

That said, shouldn't this be an error in this case?